backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/calendar/internal/BpkCalendarDayCell.kt

```diff
@@ -81,7 +85,9 @@ internal fun BpkCalendarDayCell(
                 onClickLabel = model.onClickLabel,
                 interactionSource = remember { MutableInteractionSource() },
             )
+            .testTag(if (inactive) "inactive" else "active")
```

Comment:
Is it just inactive/active that's needed as a test tag here or some more info around the date?

Reply:
<img width="266" alt="Screenshot 2024-11-27 at 16 46 32" src="https://github.com/user-attachments/assets/f58bdf0d-066b-4740-835a-3f3f80130758">


For active and inactive is mainly for detecting selectable and not selectable days. For the smoke test purpose inactive/active satisfied the need for now. If consumer need more information about the day, each day cell already contains contentdescription such as "Tuesday, 11 November 2023